WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationRemindersViewController.swift

```diff
@@ -0,0 +1,153 @@
+        formatter.doesRelativeDateFormatting = true
+        return formatter
+    }()
+
+    private lazy var noResultsViewController: NoResultsViewController = NoResultsViewController.controller()
+
+    /// Notification reminder requests displayed in the table.
+    private var requests: [UNNotificationRequest] = []
+
+    init() {
+        super.init(style: .grouped)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.title = NSLocalizedString("Notification Reminders", comment: "Title for screen showing pending reminders set for notifications")
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel,
+                                                           target: self,
+                                                           action: #selector(cancel))
+
+        ImmuTable.registerRows([ReminderRow.self, DestructiveButtonRow.self], tableView: tableView)
+
+        WPStyleGuide.configureColors(for: nil, andTableView: tableView)
+
+        reloadModel()
+    }
+
+    @objc private func cancel() {
+        dismiss(animated: true, completion: nil)
+    }
+
+    // MARK: Model and table loading
+
+    private func reloadModel() {
+        let helper = NotificationRemindersHelper()
+        helper.getPendingReminders({ requests in
+            DispatchQueue.main.async { [weak self] in
+                self?.requests = requests
+                self?.reloadTable()
+            }
+        })
+    }
+
+    private func reloadTable() {
+        let headerText = NSLocalizedString("Swipe to cancel a reminder", comment: "Instruction to user to swipe a table row to cancel the associated reminder")
+        let rows: [ReminderRow] = requests.map(row(for:))
+        let remindersSection = ImmuTableSection(headerText: headerText, rows: rows)
+
+        let title = NSLocalizedString("Cancel All Reminders", comment: "Title of button to cancel all reminders set by the user")
+        let cancelRow = DestructiveButtonRow(title: title, action: { [weak self] _ in
+            self?.helper.cancelAllReminders()
+            self?.reloadModel()
+            }, accessibilityIdentifier: "cancel reminders row")
+        let cancelAllSection = ImmuTableSection(rows: [cancelRow])
+
+        let hasReminders = requests.count > 0
+        let sections = hasReminders ? [remindersSection, cancelAllSection] : []
+
+        handler.viewModel = ImmuTable(sections: sections)
+
+        if hasReminders {
+            hideNoResults()
+        } else {
+            displayNoResults()
+        }
+    }
+
+    private func row(for request: UNNotificationRequest) -> ReminderRow {
+        let title = helper.reminderTitle(for: request)
+
+        let value: String
+        if let date = helper.reminderTriggerDate(for: request) {
+            value = formatter.string(from: date)
```

Comment:
You can use `date.shortStringWithTime()` instead and so you don't even need to define the formatter